scss/eats/customizations.scss

```diff
@@ -0,0 +1,286 @@
+        border-radius: 100px;
+        min-width: 254px;
+        margin: auto;
+
+        &:hover {
+          background-color: #25874D;
+        }
+      }
+    }
+
+    .bxi-discounts-section {
+      background-color: var(--bxi-eats-order-bg-color);
+
+      &__text-small {
+        color: var(--bxi-eats-discounts-color);
+      }
+
+      &__text-st {
+        text-decoration: line-through black; 
+      }
+
+      .bxi-discounts-row {
+        margin: 0px 70px 118px 70px;
+      }
+
+      .bxi-discounts-section-discount {
+        background-color: var(--bxi-dashboard-bg-color);
+        height: 85%;
+        border: 1px solid #E0E0E0;
+        border-radius: 8px;
+      }
+
+      .bxi-discounts-footer {
+
+        &__side {
+          display: flex;
+          flex-direction: column;
+          align-items: flex-start;
+        }
+      }
+
+      .product-img {
+        height: 164px;
+        object-fit: contain;
+      }
+
+      .bxi-button {
+        border-radius: 100px;
+      }
+    }
+
+    .bxi-footer {
+      .bxi-button {
+        border-radius: 100px;
+        border: 1px solid var(--bxi-button-bg-color);
+      }
+      .bxi-copyright-color {
+        color: var(--bxi-footer-color);
+      }
+    }
+  }
+
+  &.dashboard {
+    .bxi-eats-dashboard--color {
+      color: var(--bxi-eats-portal-color);
+    }
+    
+    .bxi-dashboard {
+      background-color: var(--bxi-eats-dashboard-container-bg-color);
+
+      .bxi-categories {
+        color: var(--bxi-button-bg-color);
+
+        .img-wrapper {
+          height: 75%;
+
+          &::before {
+            content: "";
+            display: block;
+            position: absolute;
+            top: 0;
+            left: 0;
+            width: 100%;
+            height: 50%;
+            background: linear-gradient(180deg, rgba(33, 33, 33, 0.48) 9.94%, rgba(36, 36, 36, 0) 94.52%);
+            z-index: 1;
+          }
+        }
+
+        .bxi-button {
+          border-radius: 100px;
+          color: var(--bxi-button-bg-color);
+        }
+
+      }
+
+      .bxi-eats-portal {
+        background-color: var(--bxi-dashboard-bg-color);
+
+        &--header {
+          font-size: 55px;
+          font-weight: 600;
+        }
+
+        &--text {
+          color: var(--bxi-eats-portal-checkmark-border-color);
+        }
+
+        .bxi-checkmark {
+          width: 25px;
+          min-width: 25px;
+          height: 25px;
+          border: 1px solid var(--bxi-primary-color);
+
+          svg {
+            display: none;
+          }
+
+          &--checked {
+            background-color: var(--bxi-primary-color);
+            border: none;
+
+            svg {
+              display: block
+            }
+          }
+        }
+
+        .bxi-checklist-item {
+          border-top: 1px solid #BEC9DB;
+        }
+      }
+
+      .bxi-finance-card {
+        // width: 75%;
+        max-width: 307px;
+        height: 50%;
+        margin: auto;
```

Comment:
Can you make this background a CSS variable please?

The other ones are okay since they are just shades of gray but this needs to be "re-brandable"